nsdb-http/src/main/scala/io/radicalbit/nsdb/web/routes/QueryApi.scala

```diff
@@ -98,7 +99,9 @@ case class QueryBody(@(ApiModelProperty @field)(value = "database name ") db: St
                      @(ApiModelProperty @field)(
                        value = "filters definition, adding where condition",
                        required = false,
-                       dataType = "list[io.radicalbit.nsdb.web.routes.Filter]") filters: Option[Seq[Filter]])
+                       dataType = "list[io.radicalbit.nsdb.web.routes.Filter]") filters: Option[Seq[Filter]],
+                     @(ApiModelProperty @field)(value = "return parsed query", required = false, dataType = "boolean") parsed: Option[
+                       Boolean])
```

Comment:
maybe parsed: Boolean = False instead of using an Option? But honestly I don't know any possible implication in changing this. Let's discuss it if it breaks too much

Reply:
The problem is that if we put boolean value without option, we must put in the POST request `parsed = false` even if we don't want the parsed query. And this will breaks all the "old" http requests without the parameter.

Reply:
ok. Fine. I was not completely sure about that. We can leave the option in order to maintain the backward compatibility